File: app/src/main/java/dstudio/com/retrofitrxandroid/model/userdataresponse/UserData.java

package dstudio.com.retrofitrxandroid.model.userdataresponse;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class UserData {

    @SerializedName("data")
    @Expose
    private Data data;
    @SerializedName("included")
    @Expose
    private List<Included> included = null;

    public Data getData() {
        return data;
    }

    public void setData(Data data) {
        this.data = data;
    }

    public List<Included> getIncluded() {
        return included;
    }

    public void setIncluded(List<Included> included) {
        this.included = included;
    }

}

File: app/src/main/java/dstudio/com/rxkotlinandroid/model/userdataresponse/UserData.kt
package dstudio.com.rxkotlinandroid.model.userdataresponse

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class UserData {

    @SerializedName("data")
    @Expose
    var data: Data? = null
    @SerializedName("included")
    @Expose
    var included: List<Included>? = null

}
